### ISSUE TITLE
Cannot open Model Builder UI from mbconfig. Double-clicking produces error modal.

### ISSUE THREAD

**Issue by rummelsworth** [dotnet/machinelearning-modelbuilder#1972]
**System Information (please complete the following information):**
 - Model Builder or CLI Version: 16.7.3.2143002
 - Visual Studio Version (if applicable): 17.0.4

**Describe the bug**
- On which page of the tutorial did you run into an issue (URL): https://dotnet.microsoft.com/en-us/learn/ml-dotnet/get-started-tutorial/create
- Clear description of the problem: The mbconfig file will not open. When I try, I get an error message in a modal dialog.

**To Reproduce**
1. Go to the tutorial page linked above.
2. Follow the instructions.
3. Try to open (e.g. double-click) the mbconfig file.
4. See error.

**Expected behavior**
"[...] the Model Builder UI opens in a new docked tool window in Visual Studio."

**Screenshots**

![image](https://user-images.githubusercontent.com/3069355/148588394-ad9eaac0-3f8e-4fe5-a0d0-06eaf3e7456c.png)

**Additional context**
I was just trying to follow the ML.NET tutorial.

**Comment by beccamc** [comment]
@rummelsworth Can you try updating your Model Builder to our new release - 16.9.3?

**Comment by beccamc** [comment]
@rummelsworth Sorry for the delay here! Are you still having extension problems?

**Comment by rummelsworth** [comment]
@beccamc No problem re any delay, none was perceived. (This isn't a showstopper for me by any means.)

All my other extensions work fine, including several new ones I've installed since I reported this. Model Builder 2022, despite installing again this morning without any observable problems, isn't working at all. Still the same problem reported above, with the error modal dialog appearing when I try to open the mbconfig file. Thanks.

**Comment by rummelsworth** [comment]
Any updates or advice? I realize this issue is probably an "icky" one because no one else has said they can reproduce it, haha...

I'd really like to be able to get started learning and using ML.NET. I have the latest Visual Studio (17.1.2) and the latest Model Builder 2022, and the problem is still there for me.

Is there any advice at all, beyond making sure I have the latest MB2022 extension installed? Any info or logs I can provide to help diagnose the problem?

Thanks again.

**Comment by beccamc** [comment]
@JakeRadMSFT Have we seen this before?

**Comment by beccamc** [comment]
Can you take a look at Jake's response in this issue? https://github.com/dotnet/machinelearning-modelbuilder/issues/2082

**Comment by rummelsworth** [comment]
@beccamc I tried Jake's instructions, and it didn't fix the problem.

I was able to find the MB components (plain MB as well as MB 2022) in VS Installer, and I removed them using the VS Installer. I then reinstalled using the VSIX downloaded from the extension page Jake linked. I launched VS again, opened my tutorial solution, and the same error is still happening.

**Comment by rummelsworth** [comment]
I updated, but the problem remains. I'll try to make some time soon for the reinstall.

**Comment by beccamc** [comment]
Sorry you need to take such drastic steps. I've found that a complete clean fixes this problem - https://docs.microsoft.com/en-us/visualstudio/install/uninstall-visual-studio?view=vs-2019#remove-all-with-installcleanupexe

**Comment by luisquintanilla** [comment]
@beccamc are we okay to close this issue?

**Comment by rummelsworth** [comment]
Hey y'all, I forgot to follow-up here: I actually had to use that complete clean with installcleanup.exe just last week for another extension installation problem that somehow totally destroyed my VS installation. (Again, different extension, not MB.) But, I had forgotten to try out the MB demo again to see if it would now work. I'll check this tomorrow morning and report back here.

Although, I guess maybe whether or not I'm finally fixed up might not influence whether or not y'all should close this issue. I used to be a support engineer, and this really feels like a "get a new computer" problem, lol...

Anyway, whatever y'all decide, I'll report back what happens on my end. Thanks.
